Project.toml

```
@@ -12,10 +12,11 @@ julia = "1"
 EzXML = "1"
 
 [extras]
+EzXML = "8f5d6c58-4d21-5cfd-889c-e3ad7ee6a615"
 LibGit2 = "76f85450-5226-5b5a-8eaa-529ad045b433"
 ReferenceTests = "324d217c-45ce-50fc-942e-d289b448e8cf"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [targets]
-test = ["Test", "ReferenceTests", "UUIDs", "LibGit2"]
+test = ["Test", "ReferenceTests", "UUIDs", "LibGit2", "EzXML"]
```

```suggestion
test = ["Test", "ReferenceTests", "UUIDs", "LibGit2",]
```
Not needed in test as it is a main dependency